recipes/buffer/README.md

```diff
@@ -0,0 +1,37 @@
+## buffer
+
+To make koa-better-body parse a request's body to a Buffer, you have to pass an array with accepted mime types in options.extendTypes.text Make sure to send your requests with the appropiate content-type.
+
+```js
+'use strict'
+
+var app = require('koa')()
+var path = require('path')
+var body = require('../../index')
```

Comment:
that tricky part ... reminds me and force me to create automation too for recipes, lol.

I mean, in the readme of the recipe I think it make more sense to be `koa-better-body` :)